scenes/login_dialog/login_dialog.gd

```diff
@@ -0,0 +1,25 @@
+extends Control
+
+@onready var nAnimationPlayer = $AnimationPlayer
+@onready var nLayer = $CanvasLayer/Layer
+@onready var nBox = find_child("Box")
+
+var is_opened: bool = false
+
+func _ready():
+	nLayer.visible = false
+
+func _process(delta):
+	nBox.pivot_offset.x = nBox.size.x / 2
+	nBox.pivot_offset.y = nBox.size.y / 2
+
+func open() -> void:
+	is_opened = true
+	nAnimationPlayer.play("Open")
+
+func close() -> void:
+	is_opened = false
+	nAnimationPlayer.play("Close")
+
+func _on_RegisterBtn_pressed():
+	GDialogs.open_single(GRegisterDialog)
```

Comment:
Need double white spaces between the extends statement and the variables section, between the variables section and functions, and between all functions.